templates/swagger/v1_json.tmpl

```diff
@@ -11687,6 +11687,7 @@
             "gitea",
             "gogs",
             "msteams",
+						"teamcity",
```

Comment:
Please ensure indentation is the same on this line as the others

Reply:
✔